### PR TITLE
Add autogenerated notice to docs

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Comment by dpgraham** [comment]
Fixed.

**Comment by dpgraham** [comment]
Never mind, that doesn't work either. I'll try something else.

**Comment by imurchie** [comment]
And when pushed through MkDocs?

**Comment by dpgraham** [comment]
@imurchie Yeah, I tested it locally and the comments don't show up in the rendered HTML.

**Review by jlipps — APPROVED** [review]

File: docs/en/commands/context/get-context.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands/context/get-context.yml)
```
Comment (imurchie) [inline docs/en/commands/context/get-context.md]: Should the path not be `commands-yml/commands/context/get-context.yml`?

Comment (dpgraham) [reply docs/en/commands/context/get-context.md]: Yep. Made a fix for it

**Review by imurchie — APPROVED** [review]

**Pull request merged**